lib/livebook_web/live/output/frame_component.ex

```diff
@@ -75,7 +75,7 @@ defmodule LivebookWeb.Output.FrameComponent do
     <div id={"frame-output-#{@id}"}>
       <%= if @output_count == 0 do %>
         <div class="text-gray-300 p-4 rounded-lg border border-gray-200">
-          Empty output frame
+          Nothing here
```

Comment:
```suggestion
          Nothing here...
```

Reply:
I think the `...` gives an idea of incompleteness. That something may happen or change, that this is not the final state of things. :)

Reply:
I was also considering an icon, but didn't find anything meaningful. We use the wind icon elsewhere but it looks weird here :D